kickstarts/partials/post/python_modules.ks.erb

```diff
@@ -1,7 +1,10 @@
-dnf install -y python38-pip python38-devel libcurl-devel libffi-devel libxml2-devel libxslt-devel krb5-devel openssl-devel gcc make
+# Find all intalled pythons and be sure psutil is installed in all of them
+psutils=$((for p in $(find /usr/lib -type d -name python3.* | grep -v python3.6); do basename $p | tr -d '.' | echo "$(cat -)-psutil"; done) | xargs echo)
+
+dnf install -y $psutils python38-pip python38-devel libcurl-devel libffi-devel libxml2-devel libxslt-devel krb5-devel openssl-devel gcc make
```

Comment:
If we are going to loop through all pythons I wonder if we need to install `python*-devel` as well?  Would `dnf install python39-psutil` fail if `python39-devel` weren't installed?

Reply:
No, python39-psutil is standalone and doesn't require python39-devel.

Reply:
python38-devel is only needed to build some of the eggs we install in our venv.